### ISSUE TITLE
[DONATION] OpenCloseFist

### ISSUE THREAD

**Issue by AidenRushbrooke** [uea-machine-learning/tsml_repo#97]
https://physionet.org/content/eegmmidb/1.0.0/
https://pubmed.ncbi.nlm.nih.gov/15188875/
Collected as part of a larger motor imagery archive, this dataset consists EEG readings of left vs right 
hand movements. 105 individuals took part in the experiment, completing 45 trials each, moving either their
left or right hand based on a target appearing on either the left or right side of a screen placed in front of them.
Each instance lasts 4 seconds starting 1 second before the movement began, and was recorded at 160Hz with 64 EEG channels. The first 
52 participants have been used for the training set, and the remaining 53 for the test set.
